docs/user-guide/ui/access/vpn.md

```diff
@@ -377,8 +377,9 @@ Automatically, after installing DAppNode you should see the next image and a lin
 <p align="center">
     <img src="../../../../img/VirtualBox_console.png"/>
 </p>
-
-If it does not happen, you can generate the OpenVPN credentials manually with the command:
+## I did not test this next part of the procedure; just corrected the "OpenVPN" incorrect text to "WireGuard VPN", since this is in the WireGuard section 
+If it does not happen, you can generate the WireGuard VPN credentials manually with the command:
 `dappnode_connect`
```

Comment:
This is only for openVPN. Command for wireguard is `dappnode_wireguard` this command will generate remote credentials to be copied into a wireguard client. Adding flags `--local` will make credentials for your Node's LAN and not route through a ddns. Adding flag `--qr` will generate credentials but encode it in a QR code for simple importing from command line to a phone.

Reply:
Was just flagging that the instructions in the manual that included the "OpenVPN" bits, were explicitly in the part of the document for WireGuard VPN use on Android.

Reply:
You are correct that there is a lot of incomplete and some erroneous info currently in the docs; however the whole thing is being rewritten and reformatted, hence the mistakes in copying and missing info it just hasnt been gotten to yet, the only things there are what was there before (minimal and not updated in a long time) plus a lot of new empty sections we outlined to fill as we get to it.  I'll add the whole VPN section to my to do list and finish it up.  The whole thing was written years ago before Wireguard was not even an option yet which is why openVPN is featured so prominently even where it shouldnt be.  Thanks for you contribution and pointing this out as a priority it will be handled